src/main/java/by/home/controller/CircleValidator.java

```diff
@@ -0,0 +1,15 @@
+package by.home.controller;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CircleValidator {
+
+    public Boolean isCircle(String line) {
+
+        Pattern pattern = Pattern.compile("([\\-]?[\\d]+[\\,]{1}){2}([\\-]?[\\d]+)");
+        Matcher matcher = pattern.matcher(line);
```

Comment:
Pattern pattern could be a field for performance optimization, so that it would not compile pattern each time